dev/how-to/deploy/data-migration-with-ansible.md

```diff
@@ -139,7 +139,8 @@ DM-Ansible 是 PingCAP 基于 [Ansible](https://docs.ansible.com/ansible/latest/
     wget http://download.pingcap.org/dm-ansible-{version}.tar.gz
     ```
 
-    `{version}` 为期望下载的 DM 版本，如 `v1.0.0-alpha`、`latest` 等。
+    `{version}` 为期望下载的 DM 版本，如 `v1.0.1`、`v1.0.2` 等，
```

Comment:
How about updating the `latest` used in the sample of this file. like `tar -xzvf dm-ansible-latest.tar.gz &&`.

Reply:
update in 92c930c

Reply:
update `dm_version = latest` too?

Reply:
update 9fd02b0